pkg/syslutil/setValue.go

```diff
@@ -0,0 +1,9 @@
+package syslutil
+
+func ResetVal(init string, new string) string {
```

Comment:
I dont understand what this function is supposed to be used for?

It does not reset any values, it returns the new value if and only iff the init value is empty. Function naming is very important.

Reply:
Yes, my bad, has changed name to GetNoneEmpty.